test/acceptance/integration/notifications/hadoop/hadoop_row_multi_instances.feature

```diff
@@ -31,9 +31,9 @@ Feature: start multi-instances of cygnus using hadoop sink and row mode
 
   @happy_path @multi_instances
   Scenario Outline: start multi-instances of cygnus using hadoop sink, row mode, ports differents and store multiples notifications one by instance and the port defined incremented
-    Given copy properties.json file from "epg_properties.json" to test "hdfs-sink" and sudo local "false"
+    Given copy properties.json file from "filab_properties.json" to test "hdfs-sink" and sudo local "false"
```

Comment:
I looking for multi-ways to define the properties file, but in some features this file varies by scenario. Some suggestion would be welcome. :-)